src/ImpactX.cpp

```diff
@@ -130,7 +130,7 @@ namespace impactx
 
                     // transform from x',y',t to x,y,z
                     transformation::CoordinateTransformation(*m_particle_container,
-                                                             transformation::Direction::T2Z);
+                                                             transformation::Direction::To_Fixed_T);
```

Comment:
Just a bit more in our style:
```suggestion
                                                             transformation::Direction::to_fixed_t);
```